openshift/e2e-tests-openshift.sh

```diff
@@ -27,6 +27,8 @@ readonly EVENTING_NAMESPACE="knative-eventing"
 readonly E2E_TIMEOUT="60m"
 readonly OLM_NAMESPACE="openshift-marketplace"
 
+readonly TEST_IMAGE_TEMPLATE="${IMAGE_FORMAT//\$\{component\}/knative-client-test-{{.Name}}}"
+
```

Comment:
we can refer https://github.com/openshift/knative-serving/blob/master/openshift/e2e-common.sh#L15-L25 to switch the value of this var based on the context.

> I'm not quiet sure if KO_DOCKER_REPO is needed. How will e2e tests pick up the correct image url?

@dsimansk : not needed, that should be picked up by the `--imagetemplate` flag